source/partials/_speakers.html.erb

```diff
@@ -0,0 +1,92 @@
+<section class="speakers">
+  <h3 class="speakers__title">Speakers <span>.</span></h3>
+  <div class="speakers__separator"></div>
+
+  <div class="speakers__container">
+
+    <div class="speakers__speaker">
+      <div class="speakers__photo">
+        <div class="speakers__social">
+          <a href="#" title="Facebook" class="speakers__social__media"><%= partial 'partials/svg/logo-facebook-black' %></a>
+          <a href="#" title="Twitter" class="speakers__social__media"> <%= partial 'partials/svg/logo-twitter-black' %></a>
+          <a href="#" title="Linkedin" class="speakers__social__media"> <%= partial 'partials/svg/logo-linkedin-black' %></a>
+          <a href="#" title="Instagram" class="speakers__social__media"> <%= partial 'partials/svg/logo-instagram-black' %></a>
+        </div>
+
+        <figure>
+          <img src="http://evently.mikado-themes.com/wp-content/uploads/2017/06/design-team-img-2.jpg" alt="Ada Patterson">
+        </figure>
+      </div>
+
+      <div class="speakers__description">
+        <h4 class="speakers__description__name">Palestrante Foo</h4>
+        <span class="speakers__description__role">Developer at Codeminer 42</span>
+        <p class="speakers__description__subject">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius voluptas laudantium aspernatur accusantium omnis accusamus sapiente enim ratione nam ad quibusdam, vero itaque quae natus! Quos illum quas eos voluptatibus?</p>
+      </div>
+    </div>
+
+    <div class="speakers__speaker">
+      <div class="speakers__photo">
+        <div class="speakers__social">
+          <a href="#" title="Facebook" class="speakers__social__media"><%= partial 'partials/svg/logo-facebook-black' %></a>
+          <a href="#" title="Twitter" class="speakers__social__media"> <%= partial 'partials/svg/logo-twitter-black' %></a>
+          <a href="#" title="Linkedin" class="speakers__social__media"> <%= partial 'partials/svg/logo-linkedin-black' %></a>
+          <a href="#" title="Instagram" class="speakers__social__media"> <%= partial 'partials/svg/logo-instagram-black' %></a>
```

Comment:
you can break this lines above like this:

```
<a href="#" title="Facebook" class="speakers__social__media">
  <%= partial 'partials/svg/logo-facebook-black' %>
</a>
```

Reply:
Alright! Done.